xsd/netex_framework/netex_reusableComponents/netex_seatingPlan_support.xsd

```diff
@@ -0,0 +1,353 @@
+		</xsd:annotation>
+		<xsd:simpleContent>
+			<xsd:restriction base="LocatableSpotRefStructure">
+				<xsd:attribute name="ref" type="PassengerVehicleSpotIdType" use="required">
+					<xsd:annotation>
+						<xsd:documentation>Identifier of a PASSENGER VEHICLE SPOT.</xsd:documentation>
+					</xsd:annotation>
+				</xsd:attribute>
+			</xsd:restriction>
+		</xsd:simpleContent>
+	</xsd:complexType>
+	<!-- ====== LOGGAGE SPOT ======================================= -->
+	<xsd:simpleType name="LuggageSpotIdType">
+		<xsd:annotation>
+			<xsd:documentation>Type for identifier of a LUGGAGE SPOT.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:restriction base="LocatableSpotIdType"/>
+	</xsd:simpleType>
+	<xsd:element name="LuggageSpotRef" type="LuggageSpotRefStructure" abstract="true" substitutionGroup="LocatableSpotRef">
+		<xsd:annotation>
+			<xsd:documentation>Reference to a LUGGAGE SPOT. +v2.0</xsd:documentation>
+		</xsd:annotation>
+	</xsd:element>
+	<xsd:complexType name="LuggageSpotRefStructure">
+		<xsd:annotation>
+			<xsd:documentation>Type for a reference to a LUGGAGE SPOT.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:simpleContent>
+			<xsd:restriction base="LocatableSpotRefStructure">
+				<xsd:attribute name="ref" type="LuggageSpotIdType" use="required">
+					<xsd:annotation>
+						<xsd:documentation>Identifier of a LUGGAGE SPOT.</xsd:documentation>
+					</xsd:annotation>
+				</xsd:attribute>
+			</xsd:restriction>
+		</xsd:simpleContent>
+	</xsd:complexType>
+	<!--==== TYPE OF LOCATABLE SPOT =========================================================-->
+	<xsd:simpleType name="TypeOfLocatableSpotIdType">
+		<xsd:annotation>
+			<xsd:documentation>Type for identifier of a TYPE OF LOCATABLE SPOT.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:restriction base="TypeOfValueIdType"/>
+	</xsd:simpleType>
+	<xsd:element name="TypeOfLocatableSpotRef" type="TypeOfLocatableSpotRefStructure" abstract="false" substitutionGroup="TypeOfValueRef">
+		<xsd:annotation>
+			<xsd:documentation>Reference to a TYPE OF LOCATABLE SPOT. +v2.0</xsd:documentation>
+		</xsd:annotation>
+	</xsd:element>
+	<xsd:complexType name="TypeOfLocatableSpotRefStructure" abstract="false">
+		<xsd:annotation>
+			<xsd:documentation>Type for a reference to a TYPE OF LOCATABLE SPOT.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:simpleContent>
+			<xsd:restriction base="TypeOfValueRefStructure">
+				<xsd:attribute name="ref" type="TypeOfLocatableSpotIdType" use="required">
+					<xsd:annotation>
+						<xsd:documentation>Reference to a TYPE OF LOCATABLE SPOT.</xsd:documentation>
+					</xsd:annotation>
+				</xsd:attribute>
+			</xsd:restriction>
+		</xsd:simpleContent>
+	</xsd:complexType>
+	<xsd:simpleType name="TypeOfLocatableSpotEnumeration">
+		<xsd:annotation>
+			<xsd:documentation>Allowed values for type of epassenger space.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:restriction base="xsd:normalizedString">
+			<xsd:enumeration value="seat"/>
```

Comment:
What about convertible / retractable seats that switch between seat and wheelchairSpace, seat and standingSpace, seat and pushchairSpace or bicycleSpace? Or space that can be used as any of that? 

I haven't found where the enums are used. If they could be used with cardinality *unbounded*, then it would work by specifying multiple values.